Magic/2. The Art of Magic.md

```diff
@@ -0,0 +1,32 @@
+# 2. Art of magic
+
+## The learning
+An uninitiated who don't open his spirit can't feel the praeter, even when he's near a place of power.
+Therefor, becoming a wizard is a long journey of pilgrimage to places of power to learn to open his spirit to feel and get used to the praeter. It's nothing like book reading.
+
+Once the initiated reach a certain level, he can enter the praeter properly. Then he must remain in these low bonds sources of power to progressively reach the level where he can use the praeter to cast spells.
+
+The use of strong bonds sources of power requires a way more important training and is reserved for talented wizards.
+
+The strength and the length of a spell are limited by both the caster and, if used, the source of power.
+Performing magic is exhausting for the caster, as draining energy from the praeter requires a great concentration. Although it can't injure the caster physically, it still can faint him in extreme cases.
+
+Casting does not require any gesture, lyrics or ritual.
+
+## The sources
+There are two types of sources of power: generic or specific. Generic sources allow the caster to perform any kind of magic, while specific sources allow the caster to perform only specific spells. The specifics sources of power can be seen as bridges that only open certain doors to the praeter.
+
+Eventually, the caster can perform any kind of magic when using no source of power. It's exactly like if he used a generic source of power.
+
+Any source can be either generic or specific. Here's a non-exhaustive list of examples:
+- Magical places: often places of pilgrimage for wizard novices, thus most of them allow a soft primary contact with the praeter.
+- Gemstones: diamond, amber, ruby, sapphir, amethyst, emerald, quartz, citrine, opal, etc.
+- Magical beasts: these creatures can cast spells too (?).
+
+## Bindwood
+The rosewood is a very hard and dark reddish brown type of wood. It is hard to work and takes high polish. The people of Kestal call it _bindwood_, as he has the amazing ability to drive the energy of a source of power.
```

Comment:
Right. It depends on which point of view we focus: narrative of descriptive. Although assuming they know about astrology, a generic term like _Earth_ could be necessary to compare with other planets.